frontend/src/pages/org/workflow-detail.ts

```diff
@@ -1410,6 +1428,17 @@ export class WorkflowDetail extends LiteElement {
   }
 
   private async runNow(): Promise<void> {
+    if (this.orgStorageQuotaReached === true) {
+      this.notify({
+        message: msg(
+          "The org has reached its storage limit. Delete any archived items that are un-needed to free up space, or contact us to purchase a plan with more storage."
```

Comment:
```suggestion
          "The org has reached its storage limit. Delete any archived items that are unneeded to free up space, or contact us to purchase a plan with more storage."
```
Should we put in contact info here? "contact us at _@webrecorder.net" or "contact us from the Webrecorder contact page"?

Reply:
Leaving this open. @Shrinks99 - any thoughts?

Reply:
Leaning towards not putting links in toasts?  This also seems like something that will be dealt with by including storage purchase options in the app eventually.  Additionally, hardcoding our contact info makes it vendor specific for self-hosted deployments... which maybe we don't care about considering at this time, but maybe we do?

Reply:
@Shrinks99 so maybe keep as-is for now, and come back to add this later if we see a need?